test/common_test/CMakeLists.txt

```diff
@@ -45,32 +44,30 @@ function(configure_common_test PHYSICS_ENGINE_NAME test_name)
   endif()
 endfunction()
 
-set(GZ_PHYSICS_RESOURCE_DIR "${CMAKE_SOURCE_DIR}/resources")
 
 foreach(test ${tests})
   set(test_executable "${TEST_TYPE}_${test}")
   add_executable(${test_executable} ${test}.cc)
 
   target_link_libraries(${test_executable}
     PUBLIC
-      gz-plugin${GZ_PLUGIN_VER}::loader
-      gz-common${GZ_COMMON_VER}::gz-common${GZ_COMMON_VER}
+      gz-physics-test
       ${PROJECT_LIBRARY_TARGET_NAME}
       ${PROJECT_LIBRARY_TARGET_NAME}-sdf
       ${PROJECT_LIBRARY_TARGET_NAME}-mesh
       gtest
       gtest_main
   )
 
-  target_compile_definitions(${test_executable} PRIVATE
-    "TEST_WORLD_DIR=\"${CMAKE_CURRENT_SOURCE_DIR}/worlds/\""
-    "GZ_PHYSICS_RESOURCE_DIR=\"${GZ_PHYSICS_RESOURCE_DIR}\""
-  )
+  target_include_directories(${test_executable} PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
 
   install(TARGETS ${test_executable} DESTINATION ${TEST_INSTALL_DIR})
-
   configure_common_test("bullet" ${test_executable})
   configure_common_test("bullet-featherstone" ${test_executable})
   configure_common_test("dartsim" ${test_executable})
   configure_common_test("tpe" ${test_executable})
 endforeach()
+
+if(TARGET COMMON_TEST_joint_transmitted_wrench_features)
+  target_include_directories(COMMON_TEST_joint_transmitted_wrench_features PRIVATE ${BULLET_INCLUDE_DIRS})
```

Comment:
there is a fix for this in #574; I'll see if I can split it out into its own PR

Reply:
see https://github.com/gazebosim/gz-physics/pull/591

Reply:
> see #591

#591 has been merged, so we should be able to remove this line

Reply:
Done.